cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -154,8 +180,20 @@ func startArktosNetworkController(ctx *ControllerContext, grpcHost string, grpcA
 	svcKubeClient := clientset.NewForConfigOrDie(netKubeconfigs)
 	informerFactory := externalversions.NewSharedInformerFactory(networkClient, 10*time.Minute)
 
+	// Used to create CRDs - VPC or Subnet of tenant
+	dynamicClient := dynamic.NewForConfigOrDie(netKubeconfigs)
+
+	// Used to create mapping to find out GVR via GVK
+	discoveryClient, err := discovery.NewDiscoveryClientForConfig(netKubeconfigs)
+	if err != nil {
+		klog.Errorf("when app/mizarcontrollers attempt to tart Arktos Network Controller - create discovery Client in error: (%v)", err)
```

Comment:
nit - rephrase the message - at least correct 'start' typo.

Reply:
removed it.